packages/core/stories/spinner/spinner.doc.stories.mdx

```diff
@@ -16,7 +16,7 @@ import HelpAndSupport from "docs/blocks/help-and-support.doc.mdx";
 
 # Spinner
 
-A Spinner is a visual representation of a process that's taking an indeterminate time to complete. Spinners can be triggered by automated system processes, user actions, or to indicate that content is loading. Spinners reassure the user that the system is still working, especially when the operation is taking longer than 300ms.
+A Spinner is a visual representation of a process that is taking an indeterminate time to complete. This can be triggered by automated system processes, user action, or to indicate that content is loading. This reassures the user that the system is still working, especially when the operation is taking longer than 1s.
```

Comment:
```suggestion
A Spinner is a visual representation of a process that is taking an indeterminate time to complete. This can be triggered by automated system processes, user action, or to indicate that content is loading. This reassures the user that the system is still working, especially when the operation is taking longer than one second.
```